requirements/ubuntu22.txt

```diff
@@ -0,0 +1,23 @@
+build-essential
+cmake
+pkg-config
+gcc-11
+g++-11
+clang-14
+libpython3.10-dev
+libtbb2-dev
```

Comment:
using `libtbb2-dev` (older version) instead of `libtbb-dev` (newer) to solve openvdb [conflict](https://bugs.launchpad.net/ubuntu/+source/openvdb/+bug/1970108)